esignet-service/pom.xml

```diff
@@ -29,8 +29,8 @@
 		<esignet-mock-plugin.fileName>esignet-mock-plugin.jar</esignet-mock-plugin.fileName>
 		<mosip-identity-plugin.version>1.3.0-SNAPSHOT</mosip-identity-plugin.version>
 		<mosip-identity-plugin.fileName>mosip-identity-plugin.jar</mosip-identity-plugin.fileName>
-		<esignet-digital-credential-wrapper.version>0.2.0</esignet-digital-credential-wrapper.version>
-		<esignet-digital-credential-wrapper.fileName>sunbird-rc-esignet-plugin.jar</esignet-digital-credential-wrapper.fileName>
+		<sunbird-rc-plugin.version>0.3.0-SNAPSHOT</sunbird-rc-plugin.version>
```

Comment:
if this is getting moved to esignet-plugins repo then the version should be changed to 1.3.1-SNAPSHOT. for all the plugins